akka-docs/src/main/paradox/project/rolling-update.md

```diff
@@ -1,22 +1,4 @@
-# Rolling Updates
-
-## Introduction
-
-A rolling update is the process of replacing one version of the system with another without downtime.
-The changes can be new code, changed dependencies such as new Akka version, or modified configuration.
-Rolling updates are typically used for a stateful Akka Cluster where you can't run two separate clusters in
-parallel during the update as in blue green deployments.
-
-There are two parts of Akka that need careful consideration when performing an rolling update.
-
-1. Compatibility of remote message protocols. Old nodes may send messages to new nodes and vice versa.
-1. Serialization format of persisted events and snapshots. New nodes must be able to read old data, and
-   during the update old nodes must be able to read data stored by new nodes.
-
-There are many more application specific aspects. It's important to have a strategy for serialization
-format that can be evolved and you can find advice in
-@ref:[Persistence - Schema Evolution](../persistence-schema-evolution.md), which also applies to
-remote messages when deploying with rolling updates.
```

Comment:
Relocated these to the new top level file because now we have one and it wasn't specific to migration or version updates.